notices.json

```
@@ -200,6 +200,44 @@
       }
     }
   },
+  {
+    "id": "bleve_deprecation",
+    "conditions": {
+      "audience": "sysadmin",
+      "serverVersion": ["<11.0"],
+      "displayDate": ">= 2025-10-16T00:00:00Z",
+      "serverConfig": {
+        "EnableSearching": "true",
+        "DisableDatabaseSearch": "true" }
```

I'm not super familiar with the product notices, but looking [at the code](https://github.com/mattermost/mattermost/blob/18eb1347db5dbc21ea9ab760c12efdd8a440ac56/server/channels/app/product_notices.go#L197), I think these should have the full JSON path:
```suggestion
        "BleveSettings.EnableSearching": "true",
        "SqlSettings.DisableDatabaseSearch": "true"
      }
```